blueprint/core/blueprint_View.h

```diff
@@ -79,6 +79,72 @@ namespace blueprint
         /** Invokes, if exists, the respective view event handler. */
         void dispatchViewEvent (const juce::String& eventType, const juce::var& e);
 
+         //==============================================================================
+        /** Represents a Touch object. */
+        struct Touch
+        {
+            Touch (const int identifier, const int x, const int y, const int screenX, const int screenY, const juce::var target)
+                : identifier(identifier), x(x), y(y), screenX(x), screenY(screenY), target(target)
+            {}
+
+            void setCoordinatesFromEvent(const juce::MouseEvent& e)
+            {
+                x = e.x;
+                y = e.y;
+                screenX = e.getScreenX();
+                screenY = e.getScreenY();
+            }
+
+            juce::var getVar()
+            {
+                juce::String jsonString = "{\"identifier\": " + juce::String(identifier) + ",\"x\": " + juce::String(x)+ ",\"y\": " + juce::String(y)+ ",\"screenX\": " + juce::String(screenX)+ ",\"screenY\": " + juce::String(screenY)+ ",\"target\": " + target.toString() + "}";
```

Comment:
It seems a shame to have this manual building of a JSON string here. We should be able to convert a `Touch` to a `juce::DynamicObject` instance.